src/simplewallet/simplewallet.cpp

```diff
@@ -751,6 +752,7 @@ simple_wallet::simple_wallet()
 {
   m_cmd_binder.set_handler("start_mining", boost::bind(&simple_wallet::start_mining, this, _1), tr("start_mining [<number_of_threads>] - Start mining in daemon"));
   m_cmd_binder.set_handler("stop_mining", boost::bind(&simple_wallet::stop_mining, this, _1), tr("Stop mining in daemon"));
+  m_cmd_binder.set_handler("set_node", boost::bind(&simple_wallet::set_node, this, _1), tr("Select a new remote node"));
```

Comment:
Please don't use the term "remote", that will just be confusing since it can connect to a local one too

Reply:
Removed from command bindings.